forms-flow-web/src/components/PrivateRoute.jsx

```diff
@@ -121,9 +122,18 @@ const PrivateRoute = React.memo((props) => {
       if (kcInstance) {
         authenticate(kcInstance, props.store);
       } else {
-        instance.initKeycloak(() => {
-          authenticate(instance, props.store);
-          publish("FF_AUTH", instance);
+        instance.initKeycloak((authenticated) => {
+          if(!authenticated)
+          {
+           toast.error("The user does not belong to this tenant.",{autoClose: 3000}); 
+           setTimeout(function() {
```

Comment:
need to create an unauthorized route and redirect to it. i think that's better

Reply:
The user need to send back to Login Page for updating the User creds. This flow is fine for now. THis kind of scenarios will be rare though from one tenant user Trying to login to another tenant . But similarly we need to check for if a normal keycloak user tries to login withour a designer/ reviewer / client role should the message we different say tenant key in message wont be needed there. @abilpraju-aot

Reply:
okay